setup.py

```diff
@@ -37,7 +37,6 @@ def read_version():
     "boto3>=1.13.6",
     "numpy>=1.9.0",
     "protobuf>=3.1",
-    "scipy>=0.19.0",
```

Comment:
let's create another target after line 54. maybe name is "scipy"? not sure if @nadiaya has better name ideas 😂